src/brain-even.js

```diff
@@ -0,0 +1,39 @@
+import readlineSync from 'readline-sync';
+
+console.log('Welcome to the Brain Games!');
```

Comment:
Why is it outside of a function brainEven?

Reply:
it was like a general greeting, for all games,
and when exporting a function, all the code above it - is executed,
realized, I fit it =)

Reply:
As long as I see only one game, there is no point in taking it out)

Reply:
fixed